### PR TITLE
The signal DT_SIGNAL_DEVELOP_MIPMAP_UPDATED takes an image-id as para…

### PR DESCRIPTION
**Pull request by TurboGit**
…meter.

Fix two instances passing a GList *

### CODE REVIEW

**Comment by TurboGit** [comment]
Found by chance while reading code!

File: src/common/history.c [hunk 1]
```diff
@@ -1057,8 +1057,7 @@ void dt_history_compress_on_image(const int32_t imgid)
 
   DT_DEBUG_SQLITE3_EXEC(dt_database_get(darktable.db), "COMMIT", NULL, NULL, NULL);
 
-  GList *imgs = g_list_append(NULL, GINT_TO_POINTER(imgid));
-  dt_control_signal_raise(darktable.signals, DT_SIGNAL_DEVELOP_MIPMAP_UPDATED, imgs);
+  dt_control_signal_raise(darktable.signals, DT_SIGNAL_DEVELOP_MIPMAP_UPDATED, imgid);
```
Comment (johnny-bit) [inline src/common/history.c:1060]:
```suggestion
  DT_DEBUG_CONTROL_SIGNAL_RAISE(darktable.signals, DT_SIGNAL_DEVELOP_MIPMAP_UPDATED, imgid);
```

File: src/common/history.c [hunk 2]
```diff
@@ -1113,8 +1112,7 @@ void dt_history_truncate_on_image(const int32_t imgid, const int32_t history_end
...
```
Comment (johnny-bit) [inline src/common/history.c:1115]:
```suggestion
  DT_DEBUG_CONTROL_SIGNAL_RAISE(darktable.signals, DT_SIGNAL_DEVELOP_MIPMAP_UPDATED, imgid);
```

**Review by johnny-bit — CHANGES_REQUESTED** [review]
I wonder if raise could check for wrong param types...

File: src/common/history.c (hunk 1 above)
Comment (TurboGit) [reply src/common/history.c]: Of course :)

**Comment by TurboGit** [comment]
I don't see a way to check the parameters for signals as all this is un-typed.

Merged manually. Closing.

**Pull request closed without merging**